setup.py

```diff
@@ -215,26 +230,25 @@ def src_path(self, installer: "InstallerBuildExt") -> Path:
             installer: The InstallerBuildExt instance that is installing the
                 file.
         """
-        # Share the cmake-out location with CustomBuild.
-        cmake_cache_dir = Path(installer.get_finalized_command("build").cmake_cache_dir)
+        if self.is_cmake_built:
+            # Share the cmake-out location with CustomBuild.
+            cmake_cache_dir = Path(
+                installer.get_finalized_command("build").cmake_cache_dir
+            )
+
+            cfg = get_build_type(installer.debug)
 
-        cfg = get_build_type(installer.debug)
+            if os.name == "nt":
+                # Replace %BUILD_TYPE% with the current build type.
+                self.src = self.src.replace("%BUILD_TYPE%", cfg)
+            else:
+                # Remove %BUILD_TYPE% from the path.
+                self.src = self.src.replace("/%BUILD_TYPE%", "")
 
-        if os.name == "nt":
-            # Replace %BUILD_TYPE% with the current build type.
-            self.src = self.src.replace("%BUILD_TYPE%", cfg)
+            # Construct the full source path, do not resolve globs.
+            return cmake_cache_dir / Path(self.src)
         else:
-            # Remove %BUILD_TYPE% from the path.
-            self.src = self.src.replace("/%BUILD_TYPE%", "")
-
-        # Construct the full source path, resolving globs. If there are no glob
-        # pattern characters, this will just ensure that the source file exists.
-        srcs = tuple(cmake_cache_dir.glob(self.src))
```

Comment:
Is the glob resolution not required anymore? This was important for copying python extension libraries, but maybe we stopped using it when the windows and BUILD_TYPE stuff was added.

Reply:
This is handled in line 430. `rglob()` means recursive glob resolution